packages/docs/docs/theme/inheritance.md

```diff
@@ -25,7 +25,7 @@ For now theme inheritance doesn't support high-order inheritance, that means, a
 
 ## Usage
 
-Suppose you want to create a theme inherited from the default theme, you just need to configure the [extend](./option-api.md#extend) option in your theme configuration:
+Suppose you want to create a theme inherited from the default theme, you just need to configure the [extend](./option-api.md#extend) option in your theme configuration (`.vuepress/theme/index.js`):
```

Comment:
Actually you can modify the `theme configuration` to `theme entry` and  link it to [Glossary > themeEntry](https://v1.vuepress.vuejs.org/miscellaneous/glossary.html#themeentry).